packages/migrations/src/actions/2025.12.17T00-00-00.oidc-scope.ts

```diff
@@ -0,0 +1,17 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2025.12.17T00-00-00.oidc-scope.ts',
+  noTransaction: true,
+  run: ({ sql }) => [
+    {
+      name: 'Create new "scope" column',
+      query: sql`
+        -- Alter the "oidc_integrations" table with the new column "scope"
+        ALTER TABLE "oidc_integrations"
+        ADD COLUMN "scope" TEXT[]
+        DEFAULT array['openid', 'email'];
```

Comment:
It might make sense to save only `additional_scopes` and automatically append `openid` and `email` on the application level instead of storing it in the database, as these are required for the OIDC flow.